base/Dependencies.toml

```diff
@@ -322,7 +325,7 @@ modules = [
 [[package]]
 org = "ballerinax"
 name = "health.base"
```

Comment:
This is `ballerinax/health.base` package right. Why do we have the package itself is a dependency?

Reply:
It is auto-generated. expected behaviour right?